examples/with-sw-precache/.babelrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
There's no `"test"` script in this example so the babelrc shouldn't be here right

Reply:
Yes, you are right, I corrected it.